Source/Core/DolphinWX/MainMenuBar.cpp

```diff
@@ -423,6 +423,8 @@ wxMenu* MainMenuBar::CreateSymbolsMenu() const
       IDM_SCAN_SIGNATURES, _("&Signature Database"),
       _("Recognise standard functions from Sys/totaldb.dsy, and use generic zz_ "
         "names for other functions."));
+  generate_symbols_menu->Append(IDM_SCAN_RSO, _("&RSO Modules"),
+                                _("Find functions based on RSO modules (experimental)."));
```

Comment:
Sorry for, again, missing something I should have pointed out/caught in the initial review, but the ending period should be an ellipsis, as the user still has some actions to do once they click on it (input the address). After this, it looks all good though.